app/assets/javascripts/wikis.js

```diff
@@ -8,7 +8,8 @@ function setupWiki(node_id, raw, logged_in) {
       selector: '#content-raw-markdown',
       wysiwyg: true,
       preProcessor: preProcessMarkdown,
-      postProcessor: postProcessContent
+      postProcessor: postProcessContent,
+      extraButtons: {"fa-comment": true}
```

Comment:
I see that maybe my comments are more appropriate on #1438 -- great to see you've posted that! Shall I copy this over? I'll copy over the last 3 paragraphs.